lib/_http_outgoing.js

```diff
@@ -866,6 +866,7 @@ function strictContentLength(msg) {
   );
 }
 
+let chunkBuffer = Buffer.alloc(0);
```

Comment:
Having this as a global variable will cause concurrent requests to interleave and become corrupt.